neqo-transport/src/connection.rs

```diff
@@ -1178,17 +1182,21 @@ impl Connection {
         let mut needs_padding = false;
 
         // Check whether we are sending packets in PTO mode.
-        let (pto, cong_avail, min_pn_space) =
+        let (pto, cong_avail, min_pn_space, only_not_cc) =
             if let Some((min_pto_pn_space, can_send)) = self.loss_recovery.check_pto() {
                 if !can_send {
                     return Ok(None);
                 }
-                (true, path.mtu(), min_pto_pn_space)
+                (true, path.mtu(), min_pto_pn_space, false)
             } else {
+                // If avail == 0 we do not have available congestion window, we may send only
+                // non-congestion controlled frames
+                let avail = usize::try_from(self.loss_recovery.cwnd_avail()).unwrap();
                 (
                     false,
-                    usize::try_from(self.loss_recovery.cwnd_avail()).unwrap(),
+                    if avail == 0 { path.mtu() } else { avail },
```

Comment:
I think that we want to avoid sending silly packets, so I would encourage you to find a constant greater than 0 to treat as the threshold for sending ACK-only frames.  Then you don't need to jump up to the MTU.

I note that we don't limit the size of ACK frames, so we won't respect this limit anyway...

Reply:
I put the limit to be  512 bytes.